rn/release-information/release-notes-21-08-update2.adoc

```diff
@@ -0,0 +1,76 @@
+== 21.08 Update 2 Release Notes
+
+The following table outlines the release particulars:
+
+[cols="1,4"]
+|===
+|Build
+|21.08.525
+
+|Code name
+|Iverson, 21.08 update 2
+
+|Release date
+|November 15, 2021
+
+|Type
+|Maintenance release
+
+|SHA-256 digest
+|69cdaa83c01022e5e24c4c2cf98538e9aeef08dcdae70aaf7de0d0fa2f3b43d5
+|===
+
+// Besides hosting the download on the Palo Alto Networks Customer Support Portal, we also support programmatic download (e.g., curl, wget) of the release directly from our CDN:
+//
+// LINK
+
+=== Improvements, fixes, and performance enhancements
+
+// #32709 (Compute Edition only)
+* Adds the ability for customers to use their own keys and certificates to secure the WebSocket connection between Defender and Console.
```

Comment:
Is there something specific about a websocket? Can we day to secure the connection between Defender and Console?

Reply:
@Pubs-MV I don't think it hurts to be specific. BTW, updated this description, since this mechanism lets you secure two different channels (not just one, as the requirements initially called for)